mllib/src/test/scala/org/apache/spark/ml/recommendation/ALSSuite.scala

```diff
@@ -586,6 +586,68 @@ class ALSSuite
       allModelParamSettings, checkModelData)
   }
 
+  private def checkNumericTypesALS(
```

Comment:
Is this mostly just moving the code from the test class, or did it change too?

Reply:
Mainly move but there was no mean function so implemented this oneliner.

Reply:
It's changed because one of the test hasn't done anything. Please take a look at the last commit it contains a test bugfix not only the transformation.